packages/adblocker-webextension/adblocker.ts

```diff
@@ -505,6 +572,54 @@ export class WebExtensionBlocker extends FiltersEngine {
           },
     );
   }
+
+  private executeScriptlets(browser: Browser, tabId: number, scripts: string[]): void {
+    // Dynamically injected scripts scripts can be difficult to find later in
+    // the debugger. Console logs simplifies setting up breakpoints if needed.
+    let debugMarker;
+    if (this.config.debug) {
```

Comment:
Nit. Wondering if we should reuse this flag for this purpose. On the other hand, the effect of `debug` is not documented so I guess it would not be a breaking change to use it here.

Reply:
I was only going with my general intuition what a "debug" option is: something that helps during development, but is turned off in production. If another mechanism exists to control it, we can also switch. But the debug option looks right for the job IMHO.

We could also introduce a "logging" or "verbose" option to opt-in to extra logging, which you would not want to see in production.